app/src/main/java/com/github/sdp/mediato/ui/NewItemFragment.java

```diff
@@ -51,7 +59,8 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
         if (bundle != null) {
             media = (Media) bundle.get("media");
         }
-        setItemInformation(media.getTitle(), media.getSummary(), media.getPosterUrl());
+        String summary = media.getSummary();
+        setItemInformation(media.getTitle(), summary.length() > MAX_SUMMARY_LENGTH ? summary.substring(0, MAX_SUMMARY_LENGTH) : summary, media.getPosterUrl());
```

Comment:
You could set the summary in a variable before calling setItemInformation to make it more readable.